### PR TITLE
Custom formatter documentation update

### PR DESCRIPTION
**Pull request by shagra-ms**
## Category

- [x] Content fix
- [ ] New article

## Related issues

- Custom formatter documentation update

## What's in this Pull Request?

The PR captures the following updates -
1. Added sample data for Image field 
2. `displayValue` support for Currency columns
3. Column formatting on **Currency** and **Location** columns can now be set using the modern UX
4. Support for `viewBox` and `preserveAspectRatio` for SVG
5. Support for field metadata

### CODE REVIEW

File: docs/declarative-customization/column-formatting.md [hunk 1]
```diff
@@ -1688,12 +1688,53 @@ The following example shows how a hyperlink field might be used on a current fie
...
+  "id": "6bb1d843-0633-4c9a-9a16-90bc5abd1d8e",
+  "serverRelativeUrl": "/teams/Discovery/SiteAssets/Lists/ad6ed939-0db2-4d85-8a39-8f3497f41eee/image.png",
+  "serverUrl": "https://contoso.sharepoint.com"
+}
+
+```
+
+<br/>
+
+The following example shows how a image field might be used on a current field.
```
Comment (niketjain2020) [inline docs/declarative-customization/column-formatting.md:1707]: The following example shows how an image field can be used on a current field.

Comment (shagra-ms) [reply docs/declarative-customization/column-formatting.md]: Fixed!

Comment (niketjain2020) [reply docs/declarative-customization/column-formatting.md]: good to go ahead

**Comment by andrewconnell** [comment]
Thanks for the submission. In the future, when you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.

We've done it for this PR, but please do this in the future to help speed up the process of merging community contributions. Thanks!

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**